src/core/api/reg_gc.go

```diff
@@ -50,53 +44,22 @@ func (gc *GCAPI) Prepare() {
 
 // Post ...
 func (gc *GCAPI) Post() {
-	gr := models.GCReq{}
-	gc.DecodeJSONReqAndValidate(&gr)
-	gc.submitJob(&gr)
-	gc.Redirect(http.StatusCreated, strconv.FormatInt(gr.ID, 10))
+	ajr := models.AdminJobReq{}
+	gc.DecodeJSONReqAndValidate(&ajr)
+	ajr.Name = common_job.ImageGC
+	ajr.Parameters = map[string]interface{}{
+		"redis_url_reg": os.Getenv("_REDIS_URL_REG"),
+	}
+	gc.submitAdminJob(&ajr)
+	gc.Redirect(http.StatusCreated, strconv.FormatInt(ajr.ID, 10))
 }
 
 // Put ...
```

Comment:
How about add more details in the comment to clarify what it does?  For example this only updates the schedule of the GC job.